url/src/lib.rs

```diff
@@ -73,6 +73,14 @@ assert!(data_url.fragment() == Some(""));
 # run().unwrap();
 ```
 
+## Default Features
+
+Versions `< 3` of the crate have no default features. Versions `>= 3` have the default feature 'std'.
```

Comment:
This will only be true if we really need to change version numbers

Reply:
I realized that changing the text to say `<= 2.5.2` and `> 2.5.2` so that the docs are correct regardless of whether it’s a major or minor bump.